src/services/api/api_service.cpp

```diff
@@ -18,6 +18,46 @@ namespace big
 		g_api_service = nullptr;
 	}
 
+	std::string api_service::get_translation_from_LibreTranslate(std::string message, std::string target_language)
+	{
+		std::string url = g.session.chat_translator_endpoint;
+		const auto response = g_http_client.post(url,
+		    {{"Content-Type", "application/json"}}, std::format(R"({{"q":"{}", "source":"auto", "target": "{}"}})", message, target_language));
+		if (response.status_code == 200)
+		{
+			try
+			{
+				nlohmann::json obj = nlohmann::json::parse(response.text);
+				std::string source_language = obj["detectedLanguage"]["language"];
+				std::string result = obj["translatedText"];
+
+				if (source_language == g.session.chat_translator_target && g.session.chat_translator_bypass)
+					return "";
+				return result;
+			}
+			catch (std::exception& e)
+			{
+				g_notification_service.push_error("END_KICK"_T.data(),
```

Comment:
Why is the end kick translation being used here?

Reply:
oops, idk why I dragged it over there